### DISCUSSION TITLE
Loved voting from song select

### DISCUSSION THREAD

**Discussion by mzhang28** [ppy/osu#20470] [Ideas]
This was brought up by a friend in voice chat who didn't want to submit it himself (WTF!)

Currently, loved voting must be done by clicking on the newspost either linked on the front page or the little banner at the bottom in game (in stable), and then discovering the list of beatmaps. But I've definitely come across maps (for ex. in multiplayer), that was up for loved voting and I didn't even know about it.

If there was some indicator of loved voting in the song select screen, that would increase participation in loved voting (possibly require having played ever?) and visibility for the loved candidate maps.

**Comment by peppy** [comment]
A playlist for each loved voting round could work well, as another avenue to increase exposure.
